packages_rs/nextclade/src/qc/qc_config.rs

```diff
@@ -103,6 +103,25 @@ impl Default for QcRulesConfigStopCodons {
   }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Validate)]
+#[serde(rename_all = "camelCase")]
+#[serde(default)]
+pub struct QcRulesConfigCoverage {
+  pub enabled: bool,
+  pub score_bias: f64,
+  pub score_threshold: f64,
+}
+
+impl Default for QcRulesConfigCoverage {
+  fn default() -> Self {
+    Self {
+      enabled: true,
+      score_bias: 0.0,
+      score_threshold: 100.0,
+    }
+  }
+}
+
```

Comment:
Configuration part of the coverage QC metric is here. It is what's read from `qc.json` or substituted from the `Default` trait implementation. Then it is passed to `qc::qc_config::qc_rule_coverage::rule_coverage()` and can be used there to calculate QC score and status.

Reply:
Threshold needs to be on the order of 1 to produce scores between 0 and 100 - it's 100 times too large

Reply:
Please adjust